README.md

```diff
@@ -110,6 +116,33 @@ poetry add selenium
 
 selenium --version
 
+## Working with Docker
+
+## Basic Docker Connands
+
+docker ps
+
+# Testing With Docker
+docker build --target test --tag my-test-image .
+or 
+
+docker-compose up
+docker-compose stop
```

Comment:
These commands are a bit out of place - they're not for testing with docker. Your docker-compose file is to run the app in a dev environment

Reply:
Also a common gotcha - you will usually want `docker-compose up --build`. Otherwise, it will not rebuild the image even if the Dockerfile has changed.